azure-pipelines.yml

```diff
@@ -61,6 +61,10 @@ extends:
           - repository: AppHostingSdkV4
           - repository: AndroidAppHostingSdk
           - repository: IOSAppHostingSdk
+      codeql:
```

Comment:
Why we need to disable from all-platform level? Isn't iOS only because it is the only platform using ARM64 VM?